pipelines/deployment.yml

```diff
@@ -1327,6 +1327,7 @@ jobs:
         REDIS_SERVICE: ras-redis
         UAA_CLIENT_ID: 'response_operations'
         UAA_CLIENT_SECRET: ((ci_response_operations_client_secret))
+	FEATURE_ENABLE_CLOSE_CONVERSATION: 'True'
```

Comment:
What's the red character doing there in the diff? Is that a tab character or something?. Should be fixed before it goes in.

Reply:
styling, he said hes going to fix